src/core/mesh/qgsmeshlayer.cpp

```diff
@@ -488,6 +488,12 @@ void QgsMeshLayer::assignDefaultStyleToDatasetGroup( int groupIndex )
   QgsMeshRendererScalarSettings scalarSettings;
   scalarSettings.setClassificationMinimumMaximum( groupMin, groupMax );
   scalarSettings.setColorRampShader( fcn );
+  QgsMeshStrokeWidth edgeStrokeWidth;
```

Comment:
QgsMeshStrokeWidth -> QgsMeshStrokeWidthSettings (or QgsMeshVariableStrokeWidthSettings if it does not contain fixed width)